integration/container_volume_test.go

```diff
@@ -103,19 +104,24 @@ func TestContainerSymlinkVolumes(t *testing.T) {
 			)
 
 			var (
-				testImage     = GetImage(BusyBox)
-				containerName = "test-container"
+				testImage          = GetImage(BusyBox)
+				containerName      = "test-container"
+				containerMountPath = "/mounted_file"
 			)
 
+			if goruntime.GOOS == "windows" {
+				containerMountPath = filepath.Clean("C:" + containerMountPath)
+			}
+
```

Comment:
Sorry. I suggested filepath.Clean, but can you try filepath.Abs? This will add `C:\` automatically I believe. So we can remove goruntime.OS check from here.

Reply:
filepath.Clean would add the current working directory as the prefix, which would bring us back to the issue of using a path in the container that was not in the C drive.

Reply:
Oh I see.